### ISSUE TITLE
Thanks very much..

### ISSUE THREAD

**Issue by HeyamM** [mabaorui/NeuralPull#3]
Thanks very much..
I have made as you say for the first 2 command lines, and as result, I have 3 environments, the first is base for anaconda, the second is env for the first line of command, the last one is called TensorFlow for the second command line.
Now I'm facing another problem in the third command line which is 'python setup.py build_ext --inplace', I have implemented it in the env. of the first command, is that right or I'm in the wrong direction?
the error that I have is like
![WhatsApp Image 2021-07-27 at 2 01 30 AM](https://user-images.githubusercontent.com/83015730/127070708-92105252-8f9a-44aa-a082-39018758111b.jpeg)
 the photo is attached below, could you help me, please?
thanks for your efforts.
Heyam

_Originally posted by @HeyamM in https://github.com/mabaorui/NeuralPull/issues/2#issuecomment-887088549_

**Comment by mabaorui** [comment]
You can try to download these two files https://drive.google.com/drive/folders/1t-m4iXf_XUe3yasrXjqXTYudg9LWHOhM?usp=sharing  and then run ‘python NeuralPull.py --data_dir ./exmaple_data/ --out_dir ./output_dir/ --class_idx 02691156 --train --dataset other' . If the above steps are unsuccessful，You can try 1. ‘pip install Cython’, 2. run 'python setup.py build_ext --inplace'. 3. run ‘python NeuralPull.py --data_dir ./exmaple_data/ --out_dir ./output_dir/ --class_idx 02691156 --train --dataset other'

If you still have problems, please feel free to contact me.

**Comment by HeyamM** [comment]
I have done as you said and the next issue is as follows in line 7 of setup.py ......about torch
![WhatsApp Image 2021-07-27 at 7 14 29 PM](https://user-images.githubusercontent.com/83015730/127191444-886a3617-173b-4c07-b69e-805e2b2af3b5.jpeg)
Thanks

**Comment by mabaorui** [comment]
You can try to run 'conda install -c anaconda mkl' and 'conda install -c pytorch pytorch torchvision'.

**Comment by HeyamM** [comment]
I have done your instruction and I have 2 issues the first is "no module named 'binvox_rw'' which is line 18 in NueralPull.py, also if I passed it by making it as a comment, I will have another issue in line 210 as follows:
![WhatsApp Image 2021-07-28 at 10 40 18 PM](https://user-images.githubusercontent.com/83015730/127385521-b2bb6481-bf98-4e71-b715-ee935ac9a5a1.jpeg)
Thanks for helping, I hope I don't bother you.

**Comment by mabaorui** [comment]
![image](https://user-images.githubusercontent.com/14806520/127421221-749dd927-f26b-4d66-8049-d49b7674f9eb.png)
Sorry, I forgot to upload this script to github. I have updated it. Please download it and put it in the NeuralPull path.

If you still have problems, please feel free to contact me.

**Comment by HeyamM** [comment]
Thanks very much, I have download the missed file and run the code 'python NeuralPull.py --data_dir ./exmaple_data/ --out_dir ./output_dir/ --class_idx 02691156 --train --dataset other', do I need to run 'python setup.py build_ext --inplace' before the previous running code or not? my laptop still working for hours, does it take too much time
?
![WhatsApp Image 2021-07-29 at 4 56 12 PM](https://user-images.githubusercontent.com/83015730/127506029-96f8f41d-5225-4f72-8c41-2d2944438689.jpeg)

**Comment by mabaorui** [comment]
You do not need to run 'python setup.py build_ext --inplace'.Does your program have no output staying in the screenshot interface for several hours? This result is abnormal.

**Comment by HeyamM** [comment]
Hi, mabaorui, 
I have run the code again and I'm seeing the loss of each 10 epoch. Sorry for bothering you but could I send you my email and obtain your email, if it is possible, to understand more about your work not in public like here.
my email 
hyammraht9@gmail.com
Heyam Maraha
